source/agora/test/NetworkManager.d

```diff
@@ -110,15 +110,24 @@ unittest
         public override void createNewNode (PublicKey address, Config conf)
         {
             RemoteAPI!TestAPI api;
-            if (this.nodes.length == 2)
+
+            // the test has 3 nodes:
+            // 1 validator => used for creating blocks
+            // 1 byzantine FullNode => lies about the blockchain
```

Comment:
```suggestion
            // 1 byzantine FullNode => lies about the blockchain (returns syntactically invalid data)
```

This is changing the meaning of this test a bit. Originally there were 2 validators and 1 byzantine node.
I don't mind though, but that might create issues with quorum sets.

Reply:
Actually I think the documentation is correct, it's just that it was undocumented before. Maybe this change should go into its own commit.

Reply:
Add it as own commit.